view/chart/paint.ts

```diff
@@ -28,14 +26,20 @@ function paintDot(
   width: number,
   height: number,
   background: string,
-  color: Color,
-  alpha: number
+  color: Color
 ): void {
-  if (alpha < 1) {
-    ctx.fillStyle = background
-    ctx.fillRect(x, y, width, height)
-    color.alpha = alpha
+  let bgColor = parseRgb(background)
+
+  if (!inRGB(color) && inP3(color)) {
+    color.alpha = 0.6
+    bgColor.alpha = 1 - color.alpha
+    color = blend([format(color), format(bgColor)], 'screen')
```

Comment:
You do not need to call `format()` here. `blend()` works with `Color` object from ` culori`.